envs/conda_build_config.yaml

```diff
@@ -18,6 +18,8 @@ bazel:
   - 3.7.*
 boost:
   - 1.67.0
+boost_mp11:
```

Comment:
It looks like boost_mp11 is standalone from boost-cpp, but perhaps we'll need to consider bumping up boost-cpp as well.

Reply:
Yes, I too thought so but we will have to check DALI if it is fine with newer boost. So, I left that as it is for now until we update DALI.